Vic2ToHoI4/Source/HOI4World/HoI4Country.cpp

```diff
@@ -874,4 +874,71 @@ std::optional<HoI4FocusTree> HoI4::Country::getNationalFocus() const
 	{
 		return nullopt;
 	}
+}
+
+void HoI4::Country::addSphereling(std::string Sphereling)
+{
+	double defaultAutonomy = 0.5;
+	spherelings.insert(make_pair(Sphereling, defaultAutonomy));
+}
+
+double HoI4::Country::getSpherelingAutonomy(std::string spherelingTag) const
+{
+	auto sphereling = spherelings.find(spherelingTag);
```

Comment:
In modern C++ you can move this into the if statement, limiting the scope of the new variable:
```if (auto sphereling = spherelings.find(spherelingTag); sphereling == spherelings.end())```

Do so for this and any similar checks.

Reply:
Ah, I see. It's another case of a variable used in a scope too wide, right? I'll make sure to use the above syntax.

Reply:
Yep! It's a relatively recent change that you can have both in the if statement. Before you had no choice but to have the wider scope. Note that the automatic Codacy checks will flag this as a problem. But it's just a false positive and they even have a bug open about it.